config/environment.js

```diff
@@ -52,9 +52,10 @@ module.exports = function(environment) {
     ENV['torii']['providers']['acmidm-oauth2']['baseUrl'] = 'https://authenticatie-ti.vlaanderen.be/op/v1/auth';
     ENV['torii']['providers']['acmidm-oauth2']['redirectUri'] = 'https://kaleidos-test.vlaanderen.be/authorization/callback';
     ENV['torii']['providers']['acmidm-oauth2']['logoutUrl'] = 'https://authenticatie-ti.vlaanderen.be/op/v1/logout';
+    ENV['torii']['providers']['acmidm-oauth2']['scope'] = 'vo profile openid dkbkaleidos phone'
   }
```

Comment:
With this change, you only added the change to the test environment so this would break on production
This should be higher in the torii providers (I think)
Line 28 - 38
```
    torii: {
      disableRedirectInitializer: true,
      providers: {
        'acmidm-oauth2': {
          apiKey: 'b1c78c1e-3c88-44f4-90fa-bebc5c5dc28d',
          baseUrl: 'https://authenticatie-ti.vlaanderen.be/op/v1/auth',
          scope: 'vo profile openid dkbkaleidos',
          redirectUri: 'https://kaleidos-dev.vlaanderen.be/authorization/callback',
          logoutUrl: 'https://authenticatie-ti.vlaanderen.be/op/v1/logout'
        }
      }
    }
```

Reply:
Correct, slipped the eye